core/src/main/scala/org/apache/spark/sql/streaming/WindowPhysicalPlan.scala

```diff
@@ -42,7 +42,7 @@ case class WindowPhysicalPlan(
       override def dependencies = parentStreams.toList
 
       override def slideDuration: Duration = {
-        slide.getOrElse(parentStreams.head.slideDuration)
+        parentStreams.head.slideDuration
```

Comment:
slide is no longer used?

Reply:
This was a bug reported on slack, where the parent's slide was ignored with the default one. This change fixes that bug.

Reply:
Won't this ignore the slide specified for this plan set from parser or otherwise? I think the issue to fix would be why is slide getting set to default one and not remain at None in case no explicit slide has been specified for this plan. Or does parentStreams.head somehow inherit the slide duration of this plan?